integ-test/src/test/java/org/opensearch/sql/util/TestUtils.java

```diff
@@ -166,6 +168,36 @@ private static Response retryWithoutRefreshPolicy(Request request, RestClient cl
     return client.performRequest(req);
   }
 
+  /**
+   * Compares two multiline strings representing rows of addresses to ensure they are equivalent.
+   * This method checks if the entire content of the expected and actual strings are the same. If
+   * they differ, it breaks down the strings into lines and performs a step-by-step comparison:
+   *
+   * @param expected The expected string representing rows of data.
+   * @param actual The actual string to compare against the expected.
+   */
+  public static void assertRowsEqual(String expected, String actual) {
+    if (expected.equals(actual)) {
+      return;
+    }
+
+    List<String> expectedLines = List.of(expected.split("\n"));
+    List<String> actualLines = List.of(actual.split("\n"));
+
+    if (expectedLines.size() != actualLines.size()) {
+      Assert.fail("Line count is different " + expected + " " + actual);
+    }
```

Comment:
nit: I think the message would be a bit difficult to see. Should we output like `Line count is different. expected="...", actual="...".
Same for below.